### DISCUSSION TITLE
QtCreator

### DISCUSSION THREAD

**Discussion by lrxyab** [catppuccin/catppuccin#1838] [Port Requests]
**Are you available to help in the creation of this port? 😄 (optional)**
Maybe

**Additional context 📷 (optional)**
[Other Pre-existing theme](https://github.com/ihonen/qt-creator-themes)
[Some more](https://github.com/Raincode/QtCreator-Color-Schemes)

**Comment by backwardspy** [comment]
closing as we now have a [qt creator port](github.com/catppuccin/qtcreator)!
